github-pr-reviewer/code_reviewer/github_code.py

```diff
@@ -0,0 +1,28 @@
+from github import Github
+
+from code_reviewer.reviewer import CodeReviewer
+
+class GithubAPI:
+    def __init__(self, access_token):
+        self.g = Github(access_token)
```

Comment:
Add a docstring to the __init__ method to explain the parameters and what the method does.